server/hydrogen-render/1-render-hydrogen-to-string.js

```diff
@@ -65,13 +94,20 @@ async function renderHydrogenToString(options) {
             extraErrorsMessage = ` (somehow we saw ${
               childErrors.length
             } errors but we really always expect 1 error)\n${childErrors
-              .map((childError, index) => ` ${index}. ${childError.message} ${childError.stack}`)
+              .map((childError, index) => `${index}. ${childError.stack}`)
```

Comment:
Prevent the error message from being duplicated. It's already included in the `stack`.